todo_app/app.py

```diff
@@ -1,38 +1,39 @@
 from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
 from todo_app.data.trello_items import add_new_item, delete_item, move_item, get_all_lists_and_items
-
-app = Flask(__name__)
-app.config.from_object(Config())
-
-
-@app.route('/')
-def index():
-    return render_template('index.html', lists=get_all_lists_and_items())
-
-
-@app.route('/item', methods=['POST'])
-def add_created_item():
-    if add_new_item(request.form.get('name'), request.form.get('desc'), request.form.get('due-date')) == 200:
-        return redirect('/')
-    else:
-        # Handle the error
-        pass
-
-
-@app.route("/delete_item", methods=['POST'])
-def delete_selected_item():
-    if delete_item(request.form.get('item-id')) == 200:
-        return redirect('/')
-    else:
-        # Handle the error
-        pass
-
-
-@app.route("/move_item", methods=['POST'])
-def move_item_to_new_list():
-    if move_item(request.form.get('item-id'), request.form.get('list-id')) == 200:
-        return redirect('/')
-    else:
-        # Handle the error
-        pass
+from todo_app.viewmodels import IndexViewModel
+
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+
+    @app.route('/')
+    def index():
+        return render_template('index.html', view_model=(IndexViewModel(get_all_lists_and_items())))
+
+    @app.route('/item', methods=['POST'])
+    def add_created_item():
+        if add_new_item(request.form.get('name'), request.form.get('desc'), request.form.get('due-date')) == 200:
+            return redirect('/')
+        else:
+            # Handle the error
+            pass
```

Comment:
Happy with ignoring the error handling here, but instinctively I do find the return value giving you a status code a bit odd - maybe because it means that if we were to modify this function to use a totally different data approach (e.g. a database*) then suddenly our error handling in other unconnected parts of our DB have to acknowledge that and change, or start converting DB results into status codes ourselves! Instead I'd let the function itself handle the status code and either throw errors or return some internal error type if we prefer that pattern

*Spoiler, we'll be doing exactly this later 😄 

(Absolutely happy to acknowledge this is discussion for the sake of it, no need to change unless you want to!)